src/test/java/seedu/docit/logic/commands/AddAppointmentCommandTest.java

```diff
@@ -23,17 +23,14 @@
 public class AddAppointmentCommandTest {
     private final Index defaultPatientIndex = Index.fromOneBased(1);
     private final LocalDateTime defaultDateTime = LocalDateTime.of(2020, 12, 31, 12, 0);
-    private Appointment defaultAppointment = new Appointment(SampleDataUtil.getSamplePatients()[0], defaultDateTime);
+    private Appointment defaultAppointment = new Appointment(new PatientBuilder().build(), defaultDateTime);
```

Comment:
Makes sense to use Patient Builder as the patient constructed can be customised as needed for tests 👍